styles/Header.module.css

```diff
@@ -54,6 +53,7 @@
 .searchInput {
   display: block;
   width: 100%;
+  height: 37px;
```

Comment:
@LRode this needs to be kept because the search input is not the same height as the button